apps/web/src/components/AssessmentResultsTable/types.ts

```diff
@@ -0,0 +1,32 @@
+import { ColumnDef } from "@tanstack/react-table";
+import { IntlShape } from "react-intl";
+
+import {
+  AssessmentResult,
+  AssessmentStep,
+  PoolCandidate,
+  PoolSkill,
+} from "@gc-digital-talent/graphql";
+import { IconType } from "@gc-digital-talent/ui";
+
+export type AssessmentStepResult = {
```

Comment:
I'm about to complain about a name, so I apologize... 😅 

"AssessmentStepResult" sounds to me like it holds the results for one step. But this is actually the results for ALL steps, for one skill, right?

Reply:
Though, since it includes the education result the pool skill can be undefined. 
`AssessmentTableRow` feels like the one with no caveats or exceptions.

Reply:
Okay changed to `AssessmentTableRow` in [16153aa](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9324/commits/16153aaf803acd44b522271ef501e2ca27cae510)